contracts/distribution/HAMLINKPool.sol

```diff
@@ -1,737 +0,0 @@
-/*
```

Comment:
No modifications needed after deleting this? Maybe you can undo the delete and handle deleting in another PR?